automation/build_cache

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
```

Comment:
I think this should be called `build_package_list(_cache)` or `package_info` or smth like that

Reply:
Naming is hard ;)